terraform/groups/ecs-service/data.tf

```diff
@@ -42,6 +42,15 @@ data "aws_lb_listener" "service_lb_listener" {
   port = 443
```

Comment:
Probably worth renaming "service_lb_listener" to "service_lb_listener_443", and then calling the additional listener "service_lb_listener_80" instead of "secondary_lb_listener"